### PR TITLE
Ruslan Korshunov's work

### PR DESCRIPTION
**Pull request by RuslanKorshunov**

### CODE REVIEW

File: Project (копия)/Buildings/Комплекс зданий монастыря мариавиток/building_of_church_of_st_vincent.scs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+building_of_church_of_st_vincent
```
Comment (toli444) [inline Project (копия)/Buildings/Комплекс зданий монастыря мариавиток/building_of_church_of_st_vincent.scs]: concept_building

File: Project (копия)/Buildings/Комплекс зданий монастыря мариавиток/building_of_gym_of_the_minsk_suvorov_military_school.scs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+		<-lang_ru;;
+	*);
+	[Gym of The Minsk Suvorov military school]
+	(*
+		<-lang_en;;
+	*);
+=>nrel_architect:Zaborskiy_Georgiy_Vladimirovich;
+<=nrel_subdividing:
+	{
+		building_of_housing_2_1;
```
Comment (toli444) [inline Project (копия)/Buildings/Комплекс зданий монастыря мариавиток/building_of_gym_of_the_minsk_suvorov_military_school.scs]: change identifier to some more specific

File: Project (копия)/Buildings/Комплекс зданий монастыря мариавиток/building_of_hospital_of_sisters_of_Mercy.scs [hunk 3]
```diff
@@ -0,0 +1,55 @@
+building_of_hospital_of_sisters_of_Mercy
+=>nrel_main_idtf:
```
Comment (toli444) [inline Project (копия)/Buildings/Комплекс зданий монастыря мариавиток/building_of_hospital_of_sisters_of_Mercy.scs]: concept_building

File: Project (копия)/Buildings/Комплекс зданий монастыря мариавиток/building_of_hospital_of_sisters_of_Mercy.scs [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+	(*
+		<-lang_en;;
+	*);
+	[Будынак шпіталя сясцёр міласэрнасці]
+	(*
+		<-lang_by;;
+	*);;
+
+organization_hospital_of_sisters_of_mercy
+=>nrel_main_idtf:
```
Comment (toli444) [inline Project (копия)/Buildings/Комплекс зданий монастыря мариавиток/building_of_hospital_of_sisters_of_Mercy.scs]: concept_organization

File: Project (копия)/Buildings/Комплекс зданий монастыря мариавиток/building_of_hospital_of_sisters_of_Mercy.scs [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+	organization_hospital_of_sisters_of_mercy;;
+*]
+(*
+	=>nrel_main_idtf:[Контур периода времени расположения госпиталя сестёр Милосердия в здании]
+	(*
+		<-lang_ru;;
+	*);;
+	<-begin_date_1811
+	(*
+		=>nrel_main_idtf:[1811 год]
```
Comment (toli444) [inline Project (копия)/Buildings/Комплекс зданий монастыря мариавиток/building_of_hospital_of_sisters_of_Mercy.scs]: unnecessary

File: Project (копия)/Buildings/Комплекс зданий монастыря мариавиток/building_of_hospital_of_sisters_of_Mercy.scs [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+			(*
+				<-lang_ru;;
+			*);;
+	*);;
+	<-completion_date_1854
+	(*
+		=>nrel_main_idtf:[1854 год]
```
Comment (toli444) [inline Project (копия)/Buildings/Комплекс зданий монастыря мариавиток/building_of_hospital_of_sisters_of_Mercy.scs]: unnecessary

File: Project (копия)/Buildings/Комплекс строений больницы Приказа общественного призрения/complex_of_buildings_the_minsk_hospital_of_public_charity_order.scs [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+	(*
+		<-lang_ru;;
+	*);
+	[The complex of buildings of The Minsk Hospital of Public Charity Order]
+	(*
+		<-lang_en;;
+	*);
+<-concept_complex_of_buildings;
+->building_of_psychiatry_of_minsk_hospital;
+//АВ пока разрешила так
```
Comment (toli444) [inline Project (копия)/Buildings/Комплекс строений больницы Приказа общественного призрения/complex_of_buildings_the_minsk_hospital_of_public_charity_order.scs]: remove this comment

File: Project (копия)/Cities/The_City_of_Smolensk.scs [hunk 8]
```diff
@@ -0,0 +1,29 @@
+The_City_of_Smolensk
```
Comment (toli444) [inline Project (копия)/Cities/The_City_of_Smolensk.scs]: city_Smolensk

File: Project (копия)/Date/begin_date_09_10_1924.scs [hunk 9]
```diff
@@ -0,0 +1,22 @@
+begin_date_09_10_1924
```
Comment (toli444) [inline Project (копия)/Date/begin_date_09_10_1924.scs]: We have spacific folder for this dates

File: Project (копия)/First name.scs [hunk 10]
```diff
@@ -0,0 +1,231 @@
+name_Vikientij
```
Comment (toli444) [inline Project (копия)/First name.scs]: very bad(((

Comment (toli444) [reply Project (копия)/First name.scs]: you should give very specific identifiers for all concepts

File: Project (копия)/Organization/organization_Mikhailovskij_Artillery_Academy.scs [hunk 11]
```diff
@@ -0,0 +1,15 @@
...
+	*);
+	[Mikhailovskij Artillery Academy]
+	(*
+		<-lang_en;;
+	*);
+	[Міхайлаўская артылерыйская акадэмія]
+	(*
+		<-lang_by;;
+	*);
+=>nrel_location:the_city_of_Saint_Petersburg;;
```
Comment (toli444) [inline Project (копия)/Organization/organization_Mikhailovskij_Artillery_Academy.scs]: not sure about this relation
We use it for buildings. Not for cities

File: Project (копия)/Organization/organization_the_hall_of_noble_assembly.scs [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+organization_the_hall_of_noble_assembly
+=>nrel_location:building_the_hajdukievich_house;;
+*]
+(*
+	=>nrel_main_idtf:
+		[Контур периода времени расположения зала Дворянского собрания в доме Гайдукевича]
+		(*
+			<-lang_ru;;
+		*);;
+//располагался в 1840е - 1850е
```
Comment (toli444) [inline Project (копия)/Organization/organization_the_hall_of_noble_assembly.scs]: ?

File: Project (копия)/People/Osipovich_Kazimir_Hiacintovich.scs [hunk 13]
```diff
@@ -0,0 +1,83 @@
...
+*]
+(*
+	=>nrel_main_idtf:
+		[Контур периода времени местоположения Осиповича Казимира Гиацинтовича в городе Соликамск]
+	(*
+		<-lang_ru;;
+	*);;
+	<-...
+	(*
+		=>nrel_main_idtf:
```
Comment (toli444) [inline Project (копия)/People/Osipovich_Kazimir_Hiacintovich.scs]: very bad((((

File: Project (копия)/Pieces/Пьеса "Идиллия"/piece_idilliya.scs [hunk 14]
```diff
@@ -0,0 +1,17 @@
...
+	*);
+	[Piece "Idilliya"]
+	(*
+		<-lang_en;;
+	*);
+	[Пьеса "Ідылія"]
+	(*
+		<-lang_by;;
+	*);
+->text_of_piece_idilliya;
```
Comment (toli444) [inline Project (копия)/Pieces/Пьеса "Идиллия"/piece_idilliya.scs]: Are you sure about this?

File: Project (копия)/Rrel relation/rrel_acting_rector.scs [hunk 15]
```diff
@@ -0,0 +1,13 @@
+rrer_acting_rector
+<-sc_node_role_relation;
+<-binary_relation;
+<-oriented_relation;
+=>nrel_main_idtf:
+	[и.о. ректора']
```
Comment (toli444) [inline Project (копия)/Rrel relation/rrel_acting_rector.scs]: wtf?

File: Project (копия)/Surname.scs [hunk 16]
```diff
@@ -0,0 +1,251 @@
...
+	*);
+	[Baranovskaja]
+	(*
+		<-lang_en;;
+	*);
+<-concept_surname;;
+
+surname_Uminskij
+=>nrel_main_idtf:
+	[Уминский]
```
Comment (toli444) [inline Project (копия)/Surname.scs]: baaad((

File: concept_participant_of_the_uprising_1863.scs [hunk 17]
```diff
@@ -0,0 +1,7 @@
+concept_participant_of_the_uprising_1863
```
Comment (toli444) [inline concept_participant_of_the_uprising_1863.scs]: there is no sence

File: Project (копия)/Pieces/Пьеса "Идиллия"/piece_idilliya.scs (hunk 14 above)
Comment (RuslanKorshunov) [reply Project (копия)/Pieces/Пьеса "Идиллия"/piece_idilliya.scs]: Yes

File: БЗ/Объекты местности/Комплексы строений/Комплекс зданий монастыря мариавиток/complex_of_buildings_of_the_female_mariavite_monastery.scs [hunk 18]
```diff
@@ -0,0 +1,184 @@
...
+		[Контур периода времени декомпозиции комплекса зданий монастыря мариавиток]
+		(*
+			<-lang_ru;;
+		*);;
+	<-begin_date_1811;;
+	<-completion_date_1854;;
+*);;
+
+concept_situation->[*
+	complex_of_buildings_of_the_orthodox_theological_seminary
```
Comment (toli444) [inline БЗ/Объекты местности/Комплексы строений/Комплекс зданий монастыря мариавиток/complex_of_buildings_of_the_female_mariavite_monastery.scs:72]: так не делают

**Pull request closed without merging**